repositories.bzl

```diff
@@ -155,13 +183,15 @@ def ign_bazel_repositories():
         name = "rules_python",
         strip_prefix = "rules_python-master",
         urls = ["https://github.com/bazelbuild/rules_python/archive/master.zip"],
+        sha256 = "df99e5d609b660cfa2a630bcf31f8f482b3aeb2c6ba9b915bac0ba7745a6840d"
     )
 
     _maybe(
         http_archive,
         name = "gtest",
         strip_prefix = "googletest-master",
         urls = ["https://github.com/google/googletest/archive/master.zip"],
+        sha256 = "09013320eab47ffa40a30e1ca77e08f7452945abc60dac7edbbb797f43a1d114"
```

Comment:
do not add hash when not pinning to a commit or a release? i actually have a pr for this one. same above.

Reply:
Merged from your PR.